Reference/conformance/conformance.pb.swift

```diff
@@ -88,43 +94,12 @@ enum Conformance_WireFormat: SwiftProtobuf.Enum {
     }
   }
 
-  init?(jsonName: String) {
-    switch jsonName {
-    case "UNSPECIFIED": self = .unspecified
-    case "PROTOBUF": self = .protobuf
-    case "JSON": self = .json
-    default: return nil
-    }
-  }
-
-  init?(protoName: String) {
```

Comment:
@allevato how come this is gone now? I was relying on this method myself. Any chance we can bring it back? Thanks!

Reply:
condemn. I need string value of enum type